fido_app/models.py

```diff
@@ -35,3 +38,37 @@ def set_password(self, password):
     def check_password(self, password):
         '''Checks if the given plaintext password matches the salt and hash for the given user'''
         return check_password_hash(self.password_hash, password)
+
+
+class Session(db.Model):
+    '''
+    Model associating users with persistent cookie tokens;
+    'user' property is implicitly created by relationship in User class
+    '''
+
+    id = db.Column(db.Integer, primary_key=True)
+    
+    user_id = db.Column(db.Integer, db.ForeignKey('user.id'))
+    token = db.Column(db.String(32), unique=False, nullable=False)
+    
+    interactions = db.relationship('Interaction', lazy=True, backref=db.backref('session', lazy=True))
+
+    def __repr__(self):
+        return f'<User id {self.user_id} has session token {self.token}>'
+    
+    
+class Interaction(db.Model):
+    '''
+    Model associating persistent cookie token with individual page interactions;
+    'session' property is implicitly created by relationship in Session class
+    '''
+
+    id = db.Column(db.Integer, primary_key=True)
+    
+    session_token = db.column(db.String(32), db.ForeignKey('session.token'))
```

Comment:
This should be an uppercase `Column`, right?
```suggestion
    session_token = db.Column(db.String(32), db.ForeignKey('session.token'))
```

Reply:
Yes